### ISSUE TITLE
Password fails on ssh-remote: Connect to Host...

### ISSUE THREAD

**Issue by u84six** [microsoft/vscode-remote-release#5908]
- VSCode Version: 1.62.2
- Local OS Version: Windows 10 1909
- Remote OS Version: Ubuntu 12.04
- Remote Extension/Connection Type: SSH/Docker/WSL SSH
- Logs:

Steps to Reproduce:

1. Try ssh _hostname_ from vscode terminal that uses Git Bash for Windows
2. Prompts for password and connects to host successfully
3. In vscode, run ssh-remote: Connect to Host
4. Enter _hostname_ and hit enter
5. Prompts for password, again, and then again, and again
6. Can't connect
7. At the same time, it's saying that it's installing the vscode server/service on the host, but eventually fails

Basically, it's not accepting the password for some reason. Works in terminal, not in remote-ssh

Does this issue occur when you try this locally?: Works using the terminal
Does this issue occur when you try this locally and all extensions are disabled?: Disabled all but remote-ssh and doesn't work

**Comment by tanhakabir** [comment]
Could you also post your Remote-SSH logs from the Output channel?
